x-pack/platform/plugins/shared/fleet/common/settings/agent_policy_settings.tsx

```diff
@@ -28,19 +28,20 @@ export const zodStringWithDurationValidation = z
 
 export const AGENT_POLICY_ADVANCED_SETTINGS: SettingsConfig[] = [
   {
-    name: 'agent.limits.go_max_procs',
```

Comment:
I think we may want to keep that settings around, otherwise we will introduce a breaking change to our APIs. We should probably introduce a new `deprecated` field to `SettingsConfig` that make this not visible in the UI and mark this as deprecated in the API documentation.
Some code link where we generate the API schema https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/fleet/server/services/form_settings/form_settings.ts#L21